### ISSUE TITLE
Feature Request: Provide a mechanism to display PDF content in viewer window.

### ISSUE THREAD

**Issue by warnes** [rstudio/rstudioapi#297]
RStudio can display the PDF file generated from markdown/quarto documents in the "Viewer" window, but does not appear to provide a mechanism to do so for other PDF files.

I would like to be able display existing PDF files, or files that I have created programmatically in R, within the Rstudio "Viewer" window.

Please provide a mechanisem, perhaps vie `rstudioapi::viewer()` to render PDFS in the "Viewer" window.

**Comment by jthomasmock** [comment]
Quarto render does render and then open newly created PDFs in the RStudio Viewer, but I do see that `rstudioapi::viewer()` opens in the system viewer.

Are you using Quarto or RMarkdown or wanting to open arbitrary PDF files?

**Comment by warnes** [comment]
> Are you using Quarto or RMarkdown or wanting to open arbitrary PDF files?

Yes to all three.  I have legacy RMarkdown, as well as Quarto files, I also create PDFs from HTML generated from both of these, finally I have PDFs generated elsewhere that I need to view.

**Comment by jlang-311** [comment]
@warnes thank you so much for this feature suggestion we will review this request in our future release planning to see how we could implement something like this.

**Comment by netique** [comment]
In Quarto, what you see in your viewer pane is a served webpage with pdf.js in use that renders the PDF.
